code/modules/fallout/reagents/medicines.dm

```diff
@@ -192,15 +192,15 @@ datum/reagent/medicine/super_stimpak/on_mob_life(mob/living/M)
 	..()
 	if(isliving(M))
 		to_chat(M, "<span class='notice'>The veil breaks, and the heavens spill out! The spirits of Mars float down from the heavens, and the deafining beat of the holy legion's wardrums fills your ears. Their ethereal forms are guiding you in battle!</span>")
-		M.maxHealth += 20
-		M.health += 20
+		M.maxHealth += 50
```

Comment:
So what, this is just a med-x for Legion now?

Reply:
Not quite the 70 hp or 95 hp(105 with lifegiver which is 2x the berserker powder) if you take buffout but yeah, essentially.